firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -525,8 +528,8 @@ private static void logFisCommunicationError(HttpURLConnection conn) {
   // Read the error message from the response.
   @Nullable
   private static String readErrorResponse(HttpURLConnection conn) {
-    try (BufferedReader reader =
-        new BufferedReader(new InputStreamReader(conn.getErrorStream(), UTF_8))) {
+    BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getErrorStream(), UTF_8));
```

Comment:
This can throw an IOException, yeah?
Not sure, but I think you should do the following:
```
Buffered reader = null;
try {
 reader = new BufferedReader(new InputStreamReader(conn.getErrorStream(), UTF_8));
```

Reply:
Buffer reader creation can't. It could throw null pointer exception which is handled. PTAL.